modules/aurora-mysql-resources/modules/mysql-user/main.tf

```diff
@@ -7,8 +7,8 @@ locals {
   create_db_user       = local.enabled && var.service_name != local.db_user
   save_password_in_ssm = local.enabled && var.save_password_in_ssm
 
-  db_user_key     = format("%s/%s/%s", var.ssm_path_prefix, var.service_name, "db_user")
-  db_password_key = format("%s/%s/%s", var.ssm_path_prefix, var.service_name, "db_password")
+  db_user_key     = format("%s/%s/users/%s/username", var.ssm_path_prefix, var.service_name, local.db_user)
```

Comment:
This seems needlessly redundant. is `users/foo/username` ever not going to be "foo"?

Reply:
removed db_user_key from SSM. It is redundant